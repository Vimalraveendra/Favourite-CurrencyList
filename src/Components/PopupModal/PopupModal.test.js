import React from 'react';
import{shallow} from 'enzyme';
import PopupModal from "./PopupModal"
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
const mockStore = configureStore();
const store = mockStore();

const props={}

it("should render PopupModal  component  with default props and redux store", () => {
    let wrapper = shallow(
        <Provider store={store}>
        <PopupModal {...props}/>
        </Provider>
        );
    expect(wrapper).toMatchSnapshot();
  });