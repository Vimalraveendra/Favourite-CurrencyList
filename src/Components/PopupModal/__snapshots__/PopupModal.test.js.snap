// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PopupModal  component  with default props and redux store 1`] = `ShallowWrapper {}`;
